h/api/search/query.py

```diff
@@ -97,29 +96,24 @@ def build(request_params, userid=None, search_normalized_uris=False):
     }
 
 
-def _match_clause_for_uri(uristr):
+def _filter_for_uri_match(uristr):
     """Return an Elasticsearch match clause dict for the given URI."""
```

Comment:
`"""Return an Elasticsearch filter...` this shouldn't be using match clause dicts afaict